microsoft-edge/extensions-chromium/getting-started/extension-sideloading.md

```diff
@@ -9,24 +9,44 @@ ms.date: 11/07/2022
 ---
 # Sideload an extension
 
-During development, you can use the Microsoft Edge browser to run and debug your extension safely.  By sideloading your extension locally in your browser, you can run and test your extension.  This article explains how to sideload extensions into Microsoft Edge.
+During development, you can use the Microsoft Edge browser to run and debug your extension safely.  By sideloading your extension locally in your browser, you can run and test your extension.
 
-To sideload an extension:
+To sideload an extension into Microsoft Edge:
 
-1. Open the `edge://extensions` page by choosing the three dots at the top of your browser, and then selecting **Extensions**.
+1. In Microsoft Edge, select **Settings and more** (...) > **Extensions**.
 
    ![Opening the edge://extensions page.](./media/part1-threedots.png)
 
-1. On the extension management page at `edge://extensions`, turn on **Developer mode** using the toggle at the bottom left of the page.
+   The **Extensions** pop-up box opens.
+
+1. Click **Manage extensions**.  The **Extensions** page (`edge://extensions`) opens in a new tab:
 
    ![Turning on Developer Mode.](./media/part1-developermode-toggle.png)
 
-1. When installing your extension for the first time, choose **Load Unpacked**.  You'll be prompted for the directory with your extension source files.  Your extension is installed in your browser, similar to extensions installed from the store.
+1. Turn on **Developer mode** by using the toggle at the bottom left of the page.
```

Comment:
The Developer mode toggle is at the left side on the page, but need not be at the bottom left. I just saw it is at the left but in the center. So, we should just say on the left of the page.

Reply:
simplified/standardized UI wording
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/toggle